github_test.go

```diff
@@ -20,6 +20,13 @@ const (
 
 func testGithubClient(t *testing.T) GitHub {
 	token := os.Getenv(EnvGitHubToken)
+	if token == "" {
+		if os.Getenv("CI") != "" {
+			t.Skipf("The %s environment value is not configured. skip it.", EnvGitHubToken)
+		} else {
+			t.Fatalf("The %s environment value is not configured. To skip it, set CI=true", EnvGitHubToken)
+		}
+	}
```

Comment:
We need this because secrets are not passed to the runner when a workflow is triggered from a forked repository.
See https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow